src/streams/TopicItem.js

```diff
@@ -52,15 +54,27 @@ export default function TopicItem(props: Props) {
     unreadStreams: getUnreadStreams(state),
   }));
 
+  const subscription = backgroundData.subscriptions.find(x => x.name === stream);
+
   return (
     <Touchable
       onPress={() => onPress(stream, name)}
       onLongPress={() => {
+        try {
+          if (!subscription) {
+            throw Error('Could not find subscription with given stream name.');
+          }
+        } catch (err) {
+          logging.error(err);
+          showToast(_('Something went wrong.'));
```

Comment:
Personally, I'd opt to just not show an error here — I don't think there's anything useful the user can do about this condition (which I think we expect to never happen), and just not showing the actionsheet seems better than popping up a cryptic error.